circulating-supply/src/lockup.rs

```diff
@@ -79,21 +80,17 @@ pub(super) async fn get_lockup_contract_state(
 pub(super) fn is_bug_inside_contract(
     code_hash: &near_indexer_primitives::CryptoHash,
     account_id: &near_indexer_primitives::types::AccountId,
-) -> anyhow::Result<bool> {
+) -> bool {
     match &*code_hash.to_string() {
-        // The first implementation, with the bug
-        "3kVY9qcVRoW3B5498SMX6R3rtSLiCdmBzKs7zcnzDJ7Q" => Ok(true),
-        // We have 6 lockups created at 6th of April 2021, assume it's buggy
-        "DiC9bKCqUHqoYqUXovAnqugiuntHWnM3cAc7KrgaHTu" => Ok(true),
-        // Another 5 lockups created in May/June 2021, assume they are OK
-        "Cw7bnyp4B6ypwvgZuMmJtY6rHsxP2D4PC8deqeJ3HP7D" => Ok(false),
-        // The most fresh one
-        "4Pfw2RU6e35dUsHQQoFYfwX8KFFvSRNwMSNLXuSFHXrC" => Ok(false),
-        other => anyhow::bail!(
-            "Unable to recognise the version of contract {}, code hash {}",
-            account_id,
-            other
-        ),
+        "3kVY9qcVRoW3B5498SMX6R3rtSLiCdmBzKs7zcnzDJ7Q"
+        | "DiC9bKCqUHqoYqUXovAnqugiuntHWnM3cAc7KrgaHTu" => true,
+        _ => {
```

Comment:
@telezhnaya - To avoid confusion, I've removed all contract hashes which return `false`. Now we just catch everything which isn't explicitly listed and assume it isn't buggy.

Reply:
If we have any issues with this part, we will definitely need the comments you dropped for debugging purposes. If it will be a task for the new engineer, they will suffer a lot.

I suggest leaving a comment with an explanation about all known lockup hashes (including the one you've met).

Reply:
Good point 👍🏽 I've added the contract hashes and comments back in, including the new hash.